climate_scanner/coarse_article_classifier/coarse_classifier.py

```diff
@@ -0,0 +1,34 @@
+# Imports
+
+class classifier():
+
```

Comment:
Let's consider adding an __init__ function. Here we can assign variables to hold the model in memory, and keep some configuration parameters.

Reply:
Added